src/mqt/ddsim/qasmsimulator.py

```diff
@@ -52,6 +52,7 @@ def _initialize_target(self) -> None:
 
     def __init__(self, name="qasm_simulator", description="MQT DDSIM QASM Simulator") -> None:
         super().__init__(name=name, description=description, backend_version=__version__)
+        self._simulated_circuits: list[QuantumCircuit] = []
```

Comment:
There was a reason, since for the sampler I needed direct access to the list of bound circuits. However, with the new organization of the _bind_parameters method I no longer need this access, so I removed this line.